### ISSUE TITLE
8 days after enabling 2FA new trades are still being hold in escrow 

### ISSUE THREAD

**Issue by timavo** [DoctorMcKay/node-steamcommunity#44]
Hi,

On Dec 10th I added my phone number via the Steam client to my bot's accounts and my main gaming account. For the bots I then used [this script](https://www.reddit.com/r/SteamBot/comments/3v72zz/node_small_script_to_enable_and_confirm_2fa/) based on `node-steamcommunity` to enable and confirm the Two-Factor-Authentication. On my main gaming account I used my mobile phone.

Everything worked fine on the bot's and my gaming account. I am using auth codes and confirmation keys (generated by `node-steam-totp`) for logging in and trading.

On steamcommunity.com it says "Protected by Steam Mobile Authenticator" and "Confirmation of Trades:  Enabled" on all of my accounts (all since Dec 10th).

Since yesterday, I can trade on my main gaming account without the items being held in escrow. But when trading with the bots, the items are still hold in escrow for 72 hours (even though the 7-day-period is over and all requirements for 'non-escrow' should be fulfilled).

Anyone experiencing the same issue or got any tips? Any help is appreciated!

**Comment by scholtzm** [comment]
Have you enabled trade confirmations? http://steamcommunity.com/my/edit/settings

**Comment by timavo** [comment]
Yes, I have enabled trade confirmations (like mentioned above).

I just noticed that it is working fine on one bot. On the other bot (which still has some trade on hold/escrow from the past few days) new trades are still being hold in escrow. Could it be possible that the current trades in escrow have something to do with it?

The only difference between the two bots is, that one has outstanding trades in escrow and the other doesn't.
